packages/slice-machine/test/pages/slices.test.tsx

```diff
@@ -337,13 +337,17 @@ describe("slices", () => {
       },
     });
 
-    const createOneButton = document.querySelector('[data-cy="create-slice"]');
+    const createOneButton = document.querySelector(
```

Comment:
I guess we can just getByTestId, right?

Reply:
Good catch! We already use Testing Library in this test.